### PR TITLE
chore(COVID-19): add more rooms in dungeon

### PR DESCRIPTION
**Pull request by TJXKON**
Expanded the option of room 1-D in Covid-19 Dungeon.
This is my first time contributing a project, hope to let me know if it is ok.

### CODE REVIEW

File: english/covid-19/1/1-DAA.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Althought you found a way to leave 🏃, you have met with your family  that you miss.
+
+Your family 👪 has been tested positive in Covid-19 beacause infected by you 😷!
+
+**Is it worth?**
```
Comment (Arsenic-ATG) [inline english/covid-19/1/1-DAA.md:5]:
```suggestion
**was it worth the risk?**
```

File: english/covid-19/1/1-DAB.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+You have been tested positive of Covid-19 😷.
```
Comment (Arsenic-ATG) [inline english/covid-19/1/1-DAB.md:1]:
```suggestion
You have been tested COVID-19 positive😷.
```

File: english/covid-19/1/1-DA.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+The viruses 😈 entered inside your body.
+
+**Unfortunately, you are infected by Covid-19 virus 😷!**
+
+Your coughed and felt not well 🤒.
```
Comment (Arsenic-ATG) [inline english/covid-19/1/1-DA.md:5]:
```suggestion
You coughed and felt unwell 🤒.
```

**Review by Arsenic-ATG — CHANGES_REQUESTED** [review]
some minor typo fix and rephrasing

**Comment by ming-tsai** [comment]
Hi @TJXKON,
Could you please fix this spelling problems?
image

**Comment by Arsenic-ATG** [comment]
Previous suggested changes were marked resolved by author but I can neither see a commit implementing those changes nor a reasoning for marking those as resolved, thus marking them unresolved again.

**Pull request closed without merging**